e2e/src/web/specs/timeline/utils.ts

```diff
@@ -113,13 +113,29 @@ export const thumbnailUtils = {
         ),
     ).toHaveCount(1);
   },
+  async expectThumbnailIsNotFavorite(page: Page, assetId: string) {
+    await expect(
+      thumbnailUtils
+        .withAssetId(page, assetId)
+        .locator(
+          'path[d="M12,21.35L10.55,20.03C5.4,15.36 2,12.27 2,8.5C2,5.41 4.42,3 7.5,3C9.24,3 10.91,3.81 12,5.08C13.09,3.81 14.76,3 16.5,3C19.58,3 22,5.41 22,8.5C22,12.27 18.6,15.36 13.45,20.03L12,21.35Z"]',
```

Comment:
let's use something else. I don't want random tests to break when we change the icon. why don't we simply add something like `data-test-id=favorite` to the heart icon on the thumbnail or wherever